internal/pkg/log/sanitize.go

```diff
@@ -0,0 +1,9 @@
+package log
+
+import (
+	"strings"
+)
+
+func Sanitize(in string) string {
+	return strings.TrimSpace(in)
+}
```

Comment:
Tam idem myslim o to, ze whitespaces mozu byt aj vo vnutri textu `foo\n\n\n\nbar` ... tj. trim nestaci (aj ked myslim, ze realne to hrozba velmi nie je), tj. lepsie by bolo
```
strings.Replace(in, "\n", `\n`)
```

![image](https://user-images.githubusercontent.com/19371734/193575873-f485777b-cd17-41b0-ab29-d8a1e0d7bd42.png)

Reply:
ok 🙂 👍